liberate-formula/metadata/form.yml

```diff
@@ -5,4 +5,4 @@ liberate:
      $name: 'Reinstall all packages after conversion'
      $type: boolean
      $help: 'If you want all the packages in your SLL system to have signatures from SUSE, please enable this option'
-     $default: true
+     $default: false
```

Comment:
This would not be enough, since it would only affect the ui form and not the product migration option, since it doesn't use this UI form.
The default value on this line should also be changeD to archive what you want: https://github.com/SUSE/salt-formulas/blob/master/liberate-formula/liberate/init.sls#L8

Reply:
Thanks, added to the same commit and pushed it again.